auditlog/models.py

```diff
@@ -208,6 +215,58 @@ def _get_pk_value(self, instance):
             pk = self._get_pk_value(pk)
         return pk
 
+    def _get_serialized_data_or_none(self, instance):
+        from .registry import auditlog
+
+        opts = auditlog.get_serialize_options(instance.__class__)
+        if not opts["serialize_data"]:
+            return None
+
+        model_fields = auditlog.get_model_fields(instance.__class__)
+        kwargs = opts.get("serialize_kwargs", {})
+
+        if opts["serialize_auditlog_fields_only"]:
+            kwargs.setdefault(
+                "fields", self._get_applicable_model_fields(instance, model_fields)
+            )
+
+        data = dict(json.loads(serializers.serialize("json", (instance,), **kwargs))[0])
+
+        if model_fields["mask_fields"]:
+            data = self._mask_serialized_fields(data, model_fields)
```

Comment:
It's cleaner to pass `model_fields["mask_fields"]` to this method.

Reply:
Yup yup! Done.